cmd/apiserver/drun_simulator.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	calculationsv1 "github.com/vega-project/ccb-operator/pkg/apis/calculations/v1"
+	v1 "github.com/vega-project/ccb-operator/pkg/client/clientset/versioned/typed/calculations/v1"
+	"github.com/vega-project/ccb-operator/pkg/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Simulator struct {
+	ctx        context.Context
+	fakeClient v1.VegaV1Interface
+
+	dryCalculationsTotal int
+	dryRunFailureRate    int
+	dryWorkers           int
+	dryTickerMinutes     int
+
+	dryRun bool
+	port   int
+}
+
+func gatherOptions() Simulator {
```

Comment:
why this returns a `Simulator`? Also please change the name of this func to `Bind(fs *flag.FlagSet)` and use it to bind the flags that are related to the simulator to `gatherOptions` in main.